### DISCUSSION TITLE
imgui question

### DISCUSSION THREAD

**Discussion by zhys2** [YimMenu/YimMenu#3472] [Q&A]
im making my own imgui gui, but when i add buttons and stuff and click them, the click goes through the gui and sends input to the main game, but with yimmenu when its being rendered, it blocks input to gta, how do u do that?

**Comment by xesdoog** [comment]
> im making my own imgui gui, but when i add buttons and stuff and click them, the click goes through the gui and sends input to the main game, but with yimmenu when its being rendered, it blocks input to gta, how do u do that?

You need something running in the background that constantly checks if your GUI is open and calls `PAD::DISABLE_ALL_CONTROL_ACTIONS(0)`
